lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -248,6 +248,20 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
     ></div>
 
     <div class="auth-list-container"
+      data-rolename="Group Moderator"
```

Comment:
This it the HTML for the drop down selection.  This will be update more in the UI story.